### PR TITLE
Add more logging unit test coverage

### PR DESCRIPTION
**Pull request by agv426**
Bumps the unit test coverage for logging to get develop branch coverage issue resolved

### CODE REVIEW

File: operator/internal/util/log/log_test.go [hunk 1]
```diff
@@ -25,3 +25,34 @@ func TestInitLogsDefaultInfo(t *testing.T) {
...
+	zap.S().Debugf("greeting %v", "hello")
+	msg := "InfoLevel is enabled"
+	assert.NotNil(t, zap.L().Check(zapcore.InfoLevel, msg), msg)
+	msg = "ErrorLevel is enabled"
+	assert.NotNil(t, zap.L().Check(zapcore.ErrorLevel, msg), msg)
+	msg = "DebugLevel is disabled"
+	assert.Nil(t, zap.L().Check(zapcore.DebugLevel, msg), msg)
+}
+
+func TestInitLogsDebugInfo(t *testing.T) {
```
Comment (agv426) [inline operator/internal/util/log/log_test.go:44]: Can make this a better name

Comment (bobdonat) [reply operator/internal/util/log/log_test.go:44]: @mgianatagh has a branch to increase coverage already.  I have added your additional coverage to his branch.

Comment (agv426) [reply operator/internal/util/log/log_test.go:44]: Sounds good closing this MR

**Comment by agv426** [comment]
There was already another change to add coverage, the new tests here were added into that change. Closing this one out.

File: operator/internal/util/log/log_test.go (hunk 1 above)
Comment (agv426) [reply operator/internal/util/log/log_test.go:44]: Thanks Bob, I closed this one out

**Pull request closed without merging**